Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
> destination holds collection of backups, so it is aggregating object

Backups may not be all it holds. We could have a `MirrorStreamDestination`, does `count($destination)` count the backups or the mirrors? Meh, probably just splitting hairs but if there is no great reason to have it, let's leave it out.